### ISSUE TITLE
Revise property robot.connection and extend it e.g. for Vorwerk

### ISSUE THREAD

**Issue by bjost2s** [OpenRoberta/robertalab#894]
We use currently the following connection types:
- token
- autoConnection
- arduinoAgent (deprecated)
- arduinoAgentOrToken (deprecated)

All with agent are deprecated, because we do not want to support this further.

For Vorwerk as a prototypically implementation we need something which makes it possible to connect from client directly to the robot (same network) 
e.g. local

For WeDo we may need another one, because it could communicate with a webview
e.g. bluetooth

Basically we could need more than one per robot, e.g. calliope -> autoConnection and local (via mobile app and bluetooth)

**Comment by bjost2s** [comment]
related to #893
